content/en/docs/tasks/access-application-cluster/create-external-load-balancer.md

```diff
@@ -194,6 +194,24 @@ Known issues:
 
 {{% capture discussion %}}
 
+## Load Balancer cleanup
+
+In usual case, the correlating load balancer resources in cloud provider should
+be cleaned up soon after a LoadBalancer type service is deleted. But it is known
+that there are various corner cases where resources can be leaved orphaned after
+a service is deleted. Finalizer Protection for Service LoadBalancers was
+introduced to prevent this from happening. By using finalizer, this feature
+makes sure service resource is not fully deleted until the correlating load
+balancer resources are deleted.
+
+Specifically, a finalizer with name `"service.kubernetes.io/load-balancer-cleanup"`
```

Comment:
@MrHohn do you think this could stand being written in the active voice?

You could switch to something like:
> Specifically, if a Service has a load balancer resource created for it, _component X_
> will attach a finalizer named `"service.kubernetes.io/load-balancer-cleanup"`.

I assume _component X_ will be a controller; I haven't looked up which one.